templates/kubernetes/features.html

```diff
@@ -0,0 +1,160 @@
+      <p><a class="p-button--neutral" href="/kubernetes/managed">Learn more</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <h2>Kubernetes fits perfectly on top of OpenStack, and VMware and MAAS Bare Metal</h2>
+    <p>There&rsquo;s no doubt that Kubernetes is the new standard operational layer for every multi-cloud business. Canonical enables K8s on demand for your devops teams &mdash; on OpenStack, on VMware, on public clouds, and on bare metal clusters with MAAS.</p>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <p>Deliver &lsquo;Containers as a Service&rsquo; across the enterprise with Canonical&rsquo;s Distribution of Kubernetes (CDK) which enables each project to spin up a standardised K8s of arbitrary scale, on demand, with centralised operational control.</p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Multi-cloud workload portability thanks to Ubuntu</li>
+        <li class="p-list__item is-ticked">Same Ubuntu OS and kernel as Google, Azure, AWS k8s offerings</li>
+        <li class="p-list__item is-ticked">PCI device passthrough for GPGPU, FPGA and SR-IOV workloads</li>
+        <li class="p-list__item is-ticked">AI and machine learning optimisations with nVidia and Google</li>
+        <li class="p-list__item is-ticked">Flavours to match public cloud k8s on Google, Azure, AWS</li>
+        <li class="p-list__item is-ticked">Standard upstream Kubernetes, CNCF builds available</li>
+        <li class="p-list__item is-ticked">Portfolio of third-party certified solutions for CI/CD</li>
+        <li class="p-list__item is-ticked">Day-2 operations encapsulated and included</li>
+        <li class="p-list__item is-ticked">Upgrades guaranteed between K8s releases</li>
+        <li class="p-list__item is-ticked">Security updates by Canonical just like Ubuntu</li>
+        <li class="p-list__item is-ticked">Full OCI compatibility with docker and containerd runtimes</li>
+        <li class="p-list__item is-ticked">Prometheus and ELK logging and monitoring included</li>
+        <li class="p-list__item is-ticked">Extensible, third-party ecosystem for storage, networking</li>
+      </ul>
+      <p>In partnership with Google, Canonical delivers the perfect K8s for AI and machine learning with Kubeflow.</p>
+      <p><a href="/ai/kubeflow">Learn more about Kubeflow&nbsp;&rsaquo;</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Networking</h2>
+      <p>Kubernetes imposes fundamental requirements on your networking implementation including:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">all containers can communicate with all other containers without NAT</li>
```

Comment:
Can you uppercase the first letter please.

Reply:
Done